### PR TITLE
Switch to use renderer library

### PR DESCRIPTION
**Pull request by jondewijones**
### What

Switch to use renderer library
- new config variables
- instantiate renderer library instead of renderer service
- update handler and mapper to use new library models and funcs
- update tests 

### Who can review

Anyone

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,4 +12,5 @@
 # Exclude coverage file
 coverage.txt
 
-/build
\ No newline at end of file
+/build
+assets/data.go
```
Comment (NoelleDL) [inline .gitignore:16]: New line needed here

File: Makefile [hunk 2]
```diff
@@ -29,13 +30,34 @@ lint:
...
+generate-debug: fetch-renderer-lib
+	cd assets; go run github.com/kevinburke/go-bindata/go-bindata -prefix $(CORE_ASSETS_PATH)/assets -debug -o data.go -pkg assets locales/... templates/... $(CORE_ASSETS_PATH)/assets/locales/... $(CORE_ASSETS_PATH)/assets/templates/...
+	{ echo "// +build debug\n"; cat assets/data.go; } > assets/debug.go.new
+	mv assets/debug.go.new assets/data.go
+
+.PHONY: generate-prod
+generate-prod: fetch-renderer-lib 
+	cd assets; go run github.com/kevinburke/go-bindata/go-bindata -prefix $(CORE_ASSETS_PATH)/assets -o data.go -pkg assets locales/... templates/... $(CORE_ASSETS_PATH)/assets/locales/... $(CORE_ASSETS_PATH)/assets/templates/...
+	{ echo "// +build production\n"; cat assets/data.go; } > assets/data.go.new
+	mv assets/data.go.new assets/data.go
```
Comment (NoelleDL) [inline Makefile:63]: New line here too

File: config/config_test.go [hunk 3]
```diff
@@ -33,6 +32,7 @@ func TestConfig(t *testing.T) {
 				So(cfg.DefaultLimit, ShouldEqual, 10)
 				So(cfg.DefaultMaximumLimit, ShouldEqual, 50)
 				So(cfg.DefaultMaximumSearchResults, ShouldEqual, 500)
+				So(cfg.SiteDomain, ShouldEqual, "localhost")
 			})
```
Comment (NoelleDL) [inline config/config_test.go:36]: Should there be tests for the other new config variables? `Debug` and `SupportedLanguages`?

File: Makefile (hunk 2 above)
Comment (jondewijones) [reply Makefile]: 👍

File: config/config_test.go (hunk 3 above)
Comment (jondewijones) [reply config/config_test.go:38]: 👍

File: .gitignore (hunk 1 above)
Comment (jondewijones) [reply .gitignore]: 👍

**Review by rav-pradhan — APPROVED** [review]

**Pull request merged**